### PR TITLE
Dockerize 

### PR DESCRIPTION
**Pull request by mrcnc**
## Status
**READY**

## JIRA Ticket
SPACON-350

## Description
Replaces the node server and updates the docker configuration for local development

## Todos
- [x] Tests
- [x] Documentation
- [x] License

### CODE REVIEW

File: server/src/spacon/components/mqtt.clj [hunk 1]
```diff
@@ -9,8 +9,6 @@
             [clojure.core.async :as async]
             [spacon.http.auth :refer [get-token]]))
 
-(def id "spacon-server")
```
Comment (mrcnc) [inline server/src/spacon/components/mqtt.clj]: keep this as `"spacon-server"` for the client id

File: server/src/spacon/http/auth.clj [hunk 2]
```diff
@@ -35,12 +36,19 @@
 (defn authorize-user
   ;; Currently, this is used by the mqtt broker to ensure that only authenticated users are able to connect to the
   ;; broker.  Eventually we will want to use this handler to ensure that a user has permission to subscribe or publish
-  ;; to a specific topic
+  ;; to a specific topic.
   [request]
-  (response/ok))
+  (println request)
```
Comment (mrcnc) [inline server/src/spacon/http/auth.clj]: remove println

File: server/test/spacon/proto_test.clj [hunk 3]
```diff
@@ -0,0 +1,12 @@
+(ns spacon.proto-test
```
Comment (mrcnc) [inline server/test/spacon/proto_test.clj]: delete this file

**Comment by tetriscode** [comment]
👍

**Pull request merged**